OWNERS_ALIASES

```diff
@@ -1,13 +1,82 @@
 # See the OWNERS_ALIASES docs: https://git.k8s.io/community/contributors/guide/owners.md#OWNERS_ALIASES
-# Do not change indents. Incorrect indents break the Prow CI
+# Do not change indents
+# Incorrect indents break the Prow CI
 
 aliases:
-  merge-rights: {}
-  approve-rights:
+  peer-reviewers-a:
+  - bergerhoffer
+  - stevsmit
+  - snarayan-redhat
+  - JoeAldinger
+  - michaelryanpeter
+  - skrthomas
+  - GroceryBoyJr
+  - eohartman
+  - agantony
+  - mletalie
+  - mramendi
+  - aspauldi
+  - abrennan89
+  - mpandya15
+  peer-reviewers-b:
+  - bmcelvee
+  - mburke5678
+  - jldohmann
+  - kcarmichael08
+  - ousleyp
+  - opayne1
+  - Srivaralakshmi
+  - dfitzmau
+  - sabrinajess
+  - cbippley
+  - kowen-rh
+  - jmanthei
+  - max-cx
+  peer-reviewers-c:
+  - adellape
+  - lpettyjo
+  - jeana-redhat
+  - sheriff-rh
+  - bscott-rh
+  - skopacz1
+  - gaurav-nelson
+  - aldiazRH
+  - tmalove
+  - xenolinux
+  - gwynnemonahan
+  - smunje1
+  - obrown1205
+  peer-reviewers-d:
+  - jab-rh
+  - maxwelldb
+  - EricPonvelle
+  - sjhala-ccs
+  - bburt-rh
+  - aireilly
+  - lahinson
+  - ShaunaDiaz
+  - DCChadwick
+  - shipsing
+  - rh-tokeefe
+  - bhardesty
+  - jneczypor
+  merge-reviewers:
```

Comment:
I don't know how this will work exactly but should these also be broken up by squads? So only mergers on squad A get merge pings during A week?

Reply:
It will work by creating a PR against OWNERS to comment out squads. For example, one week:

```makefile
reviewers:
- merge-rights
- peer-reviewers-a
# - peer-reviewers-b
# - peer-reviewers-c
# - peer-reviewers-d
```
Next week:

```makefile
# - peer-reviewers-a
- peer-reviewers-b
# - peer-reviewers-c
# - peer-reviewers-d
```

Reply:
Whatever team is uncommented under `reviewers:` in OWNERS gets the round-robin assignment. Reviewers are free to unassign themselves from the review which puts it back in the pool

Reply:
I broke it out a bit. `reviewers` includes anyone in a peer review squad alias. `merge-reviewers` are just folks with merge rights - these are the "approvers".

Reply:
IN a busy week, say before merge freeze, you could do (for example):

```
reviewers:
- peer-reviewers-a
- peer-reviewers-b
- peer-reviewers-c
- peer-reviewers-d
```

Reply:
`OWNERS` is where you control the actual review assignments. `OWNER_ALIASES` is just the pool of team members.

```
# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
# This file uses aliases defined in OWNERS_ALIASES
# Do not change indents
# Incorrect indents break the Prow CI

# Uncomment to enable review squads
reviewers:
- peer-reviewers-a
# - peer-reviewers-b
# - peer-reviewers-c
# - peer-reviewers-d

approvers:
- merge-reviewers
```

Reply:
@aireilly Did you answer @jeana-redhat question? Will merge reviewers be assigned PRs for merge review while not "officially" in merge review duty? 

Also, is this an exhaustive list of merge reviewers? If so, I am not in the list.

Reply:
> Also, is this an exhaustive list of merge reviewers? If so, I am not in the list.

Updated. This was based on what was previously in GH, though no functionality was assigned to the list.

@kalexand-rh @mburke5678 I believe we did not discuss the merge rights approvers assignment. I guess we could match it to the peer review list or keep it as a big group like it is set in this PR.